### PR TITLE
Alerts importing complete, closes #1

### LINKED ISSUE #1: Alerts backup/restore
I'd like to be able to migrate my alerts from one Cloud Manager (or Ops Manager) group to another. Maybe keep a local copy for initializing new groups, too. Suggested format:

```
mops alerts export GROUP_ID FILE
```

This will take the stored credentials, export `GROUP_ID`'s alerts into some file format at `FILE`.

Then I want to be able to restore

```
mops alerts restore GROUP_ID FILE
```

This will take the stored credentials, take the alerts stored in `FILE` and import/restore them to `GROUP_ID`. Using `mops login` in between these two commands would allow me to migrate alert configurations from Cloud Manager to Ops Manager, vice versa, or between Ops Manager installs, or simply change API keys if I want to. That is, nothing in `FILE` should be specific to the particular group or environment.

### PR DESCRIPTION
**Pull request by PeaSeaGee**
This now takes a JSON file of alerts and imports them into a group. This can be used to copy alerts from one group to the next.

### CODE REVIEW

File: bin/mops-alertsImport [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    var mms = new MMS({
+        username: user,
+        apiKey: apiKey,
+        host: host
+    });
+    inputConfig['results'].forEach(function(alert) {
+        delete alert.created;
+        delete alert.updated;
+        delete alert["id"];
+        delete alert.links;
```
Comment (denniskuczynski) [inline bin/mops-alertsImport]: [np] maybe break up this function a bit?  `createAlert(removeUnecessaryFields(alert));

File: bin/mops-alertsImport [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        alert.groupId = groupId;
+        console.log('current alert: ',JSON.stringify(alert,null,1))
+        var insertAlert = mms.post('/groups/'+groupId+'/alertConfigs', alert, function(err, response) {
+            console.log('Error Posting alert config: ', err);
+            console.log('Response Posting alert config: ', response);
+        });
+        q.all(insertAlert).then(
```
Comment (denniskuczynski) [inline bin/mops-alertsImport:67]: We could also 

```
var allInserts = inputConfig['results'].map(function(alert) {
    ...
    return insertAlert;
});
q.all(allInserts).then(
    function() {  console.log('all inserted');
});
```

**Comment by denniskuczynski** [comment]
LGTM

File: bin/mops-alertsImport (hunk 1 above)
Comment (PeaSeaGee) [inline bin/mops-alertsImport]: No idea what [np] means, but I'm doing this now.

Comment (denniskuczynski) [inline bin/mops-alertsImport]: Sorry - nitpick :-)

**Pull request merged**